test/Vegetation/canopy_model.jl

```diff
@@ -53,6 +53,7 @@ include(joinpath(pkgdir(ClimaLSM), "parameters", "create_parameters.jl"))
         return FT(
             instantaneous_zenith_angle(
                 DateTime(t),
+                Insolation.OrbitalData(),
```

Comment:
`Insolation.OrbitalData()` is an expensive call, so you may want to pass this variable in (let's call in `orbital_data`) if this function is called in a loop or across points in space.